pkg/handler/v1/provider/openstack.go

```diff
@@ -28,34 +28,44 @@ import (
 	"k8c.io/kubermatic/v2/pkg/provider"
 )
 
+type credential struct {
```

Comment:
nit: I would rather call this `credentials`, you could also consider using OpenstackReq directly instead of defining a new struct

Reply:
will change it to credentials for the consistency